web-console/future-openshift/cron-jobs/cron-jobs.md

```diff
@@ -0,0 +1,27 @@
+# Cron Jobs
+
+## Create Cron Job
+![](img/cron-job-list.png)
+- Clicking the `Create Cron Job` button launches the creation form.
+
+![](img/create-cron-job.png)
+- Users are prompted to enter a name for the Cron Job, a schedule, and a job template to run.
+- By default, users are presented with a cron expression field and a link to documentation explaining the syntax.
+
+![](img/create-cron-job-2.png)
+- Alternatively, users may select a period which presents the relevant fields for stating when the report should be run
+
+## Suspend Cron Job
+![](img/cron-job-suspend.png)
+- A cron job can be run immediately by clicking the `Run Job` button.
+- Jobs can also be temporarily suspended by selecting `Suspend Jobs` from the actions menu.
+
+![](img/cron-job-suspended.png)
+- When cron job has been suspended, a banner appears at the top of the page explaining the state and proving a button to resume normal activity.
+
+![](img/cron-job-resume.png)
```

Comment:
same question ... if this is just resuming the current job, should it just be called Resume Job?

Maybe i'm misunderstanding though.  Is this resuming all jobs associated with this cron job ?

Reply:
See above ^^